project/First-Project/src/main/java/first/project/controller/AuthenticationController.java

```diff
@@ -53,4 +62,22 @@ public String processRegister(User user, Model model)
             return "register";
         }
     }
-}
\ No newline at end of file
+
+    @PostMapping("/login-successful")
+    @ResponseBody
+    public ResponseEntity<String> processLogin(@RequestBody UserDTO userDTO)
+    {
+        User user = new User();
+        user.setPassword(userDTO.getPassword());
+        user.setEmail(userDTO.getEmail());
+        ArrayList<String> res = authenticationService.validateAndLogin(user);
```

Comment:
Send UserDTO instead of user. There is no need to create user object.
Change method signature to accept UserDTO instead of User and send the DTO object